examples/launch_templates/main.tf

```diff
@@ -64,10 +64,19 @@ module "eks" {
       public_ip            = true
     },
     {
-      name                 = "worker-group-2"
+      name                 = "worker-spot-1"
```

Comment:
Can you leave this example as is but change the one in `examples/spot_instances` instead?

Reply:
I will change it in two days as I am quite fur from good internet :)

Reply:
I have made the changes